alexthompson207/current-climate
# Feature/favorites

# PR CurrentClimate

### What does this do?


### Are there any known bugs?


### Summary of the changes
Create a view for the user to view their favorite articles. Implement local storage so favorites persist on pageload. Add styling to the Favorites view and create a button in the nav to view favorites. 

### What ticket(s) do the changes resolve?


Please Link Issues to this PR as necessary.

### Notes for the Reviewer
N/A


### How to Test Changes?
Clone or fetch the branch down. Click on the icon heart icon to favorite an article. Click on the favorites tab to view all favorites. Favorites should persist on page load.

### Next Iterations
Test favorites view
